src/lifetracker/command/EditRecurringEntryCommand.java

```diff
@@ -13,11 +13,11 @@
 public class EditRecurringEntryCommand extends EditGenericTaskCommand {
 
     static final int OCCUR_INF = -1;
-    protected static final int OCCUR_DATE = -2;
+    static final int OCCUR_DATE = -2;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/2007-yichen/main/file/2384997634/issues/source?bid=3226269&fileBranchId=3227675#l16)